Ladybird/Tab.cpp

```diff
@@ -170,6 +159,12 @@ void Tab::navigate(QString url)
     view().load(ak_deprecated_string_from_qstring(url));
 }
 
+void Tab::load(URL const& url, LoadType load_type)
```

Comment:
Now we have `navigate(url)` and `load(url, LoadType::HistoryNavigation)` and it's not at all clear what the difference is. Let's not do that.